lib/chef/provisioning/aws_driver/aws_resource.rb

```diff
@@ -5,6 +5,9 @@
 # Common AWS resource - contains metadata that all AWS resources will need
 module Chef::Provisioning::AWSDriver
 class AWSResource < Chef::Provisioning::AWSDriver::SuperLWRP
+  actions :create, :destroy, :nothing
```

Comment:
John and I talked about it in length. I'd be happy to give you a run down
tommorow.

On Thursday, March 19, 2015, Tyler Ball <notifications@github.com> wrote:

> In lib/chef/provisioning/aws_driver/aws_resource.rb
> <https://github.com/chef/chef-provisioning-aws/pull/136#discussion_r26810028>
> :
>
> > @@ -5,6 +5,9 @@
> >  # Common AWS resource - contains metadata that all AWS resources will need
> >  module Chef::Provisioning::AWSDriver
> >  class AWSResource < Chef::Provisioning::AWSDriver::SuperLWRP
> > +  actions :create, :destroy, :nothing
>
> How will this work with :attach and :detach for @patrick-wright
> <https://github.com/patrick-wright>'s ebs_volume PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chef/chef-provisioning-aws/pull/136/files#r26810028>.
>